<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="winChanger" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>891</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>2</width>
      <height>2</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>901</width>
        <height>571</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>450</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>16</y>
         <width>741</width>
         <height>541</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="QLabel" name="WinLabel">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>260</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="score1">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>30</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="score2">
       <property name="geometry">
        <rect>
         <x>769</x>
         <y>132</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="playerTurn">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>360</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>PlayerTurn</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="AITimeoutBox">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>530</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="AILabel">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>500</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>AI Timeout</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>531</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="PvP">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>240</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Player vs Player</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PvE">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>330</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Player vs IA</string>
       </property>
      </widget>
      <widget class="QLabel" name="name">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>491</width>
         <height>201</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>8</horstretch>
         <verstretch>8</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>URW Palladio L</family>
         <pointsize>42</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="lineWidth">
        <number>4</number>
       </property>
       <property name="text">
        <string>Gomoku !</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>name</cstring>
       </property>
      </widget>
      <widget class="modifiableQLabel" name="Message">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>421</width>
         <height>211</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="doubleThree">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>420</y>
         <width>251</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Purisa</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Double Trois</string>
       </property>
       <property name="iconSize">
        <size>
         <width>125</width>
         <height>125</height>
        </size>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="eatThem">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>440</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Purisa</family>
         <pointsize>26</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Mange Pions</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="ErrorMessage">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>584</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kinnari</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionChangeName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>changeName</string>
   </property>
   <property name="toolTip">
    <string>change the name</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>modifiableQLabel</class>
   <extends>QLabel</extends>
   <header location="global">modifiableQLabel.hpp</header>
   <slots>
    <slot>setQLabel()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>winChanger</class>
   <extends>QStackedWidget</extends>
   <header location="global">winchanger.hpp</header>
   <container>1</container>
   <slots>
    <slot>nextWidget()</slot>
    <slot>prevWidget()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>PvE</sender>
   <signal>clicked()</signal>
   <receiver>frame_2</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>frame</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PvP</sender>
   <signal>clicked()</signal>
   <receiver>stackedWidget</receiver>
   <slot>nextWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>stackedWidget</receiver>
   <slot>prevWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PvE</sender>
   <signal>clicked()</signal>
   <receiver>stackedWidget</receiver>
   <slot>nextWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
